galaxy_ng/app/api/ui/urls.py

```diff
@@ -31,6 +31,21 @@
     path("logout/", views.LogoutView.as_view(), name="auth-logout"),
 ]
 
+container_paths = [
+    path(
+        "repositories/",
+        viewsets.CotainerRepositoryViewSet.as_view({'get': 'list'}),
+        name='container-repository-detail'),
+    path(
+        "repositories/<str:name>/",
+        viewsets.CotainerRepositoryViewSet.as_view({'get': 'retrieve'}),
+        name='container-repository-detail'),
+    path(
+        "repositories/<str:namespace>/<str:name>/",
```

Comment:
these 2 path options... are they both needed because a user may enter name in these 2 formats? `newswanger/ubuntu` or `"newswanger/ubuntu"`?

Reply:
yeah, `foo` and `foo/bar` are both valid image names. I'm not sure how to capture this with just one route since it contains `/`. Any idea on how to do this is appreciated.

Reply:
given it may or may not have a `/` this seems reasonable